<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>890</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="start_date">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>13</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="start_date_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>대회 시작일 :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="today_label">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>15</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>기준일 :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDateEdit" name="last_date">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>13</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>13</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>저 장</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_button">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>13</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>시 세 조 회</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_rank_button">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>13</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>순 위 갱 신</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>42</y>
      <width>1101</width>
      <height>821</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="load_from_excel_button">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>13</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load xlsx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="screenshot_button">
    <property name="geometry">
     <rect>
      <x>684</x>
      <y>13</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Take screenshot</string>
    </property>
   </widget>
   <widget class="QLabel" name="avg_ration">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>15</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>평균 수익률 : </string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ratiotextBrowser">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>13</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1122</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
